controllers/cluster_controller.go

```diff
@@ -164,6 +168,32 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *clusterv1.Cl
 	return res, kerrors.NewAggregate(errs)
 }
 
+func (r *ClusterReconciler) reconcileMetrics(_ context.Context, c *clusterv1.Cluster) {
```

Comment:
```suggestion
func (r *ClusterReconciler) reconcileMetrics(_ context.Context, cluster *clusterv1.Cluster) {
```

for consistency with the other methods